java/src/apps/util/Log4JProblemReportProvider.java

```diff
@@ -1,12 +1,15 @@
 package apps.util;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Enumeration;
+import java.util.*;
 
 import jmri.util.problemreport.LogProblemReportProvider;
 
-import org.apache.log4j.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Appender;
+import org.apache.logging.log4j.core.appender.FileAppender;
+import org.apache.logging.log4j.core.Logger; // not org.apache.logging.log4j.Logger
```

Comment:
What's the difference between these two?

Reply:
The vast majority of the time on Log4J2 it's `org.apache.logging.log4j.Logger` to access the Logger.
To get the Appenders though, we need to use `org.apache.logging.log4j.core.Logger`